tools/tests/com.liferay.ide.gradle.core.tests/src/com/liferay/ide/gradle/core/tests/NewLiferayWorkspaceOpTests.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.gradle.core.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import com.liferay.ide.core.util.CoreUtil;
+import com.liferay.ide.gradle.core.workspace.NewLiferayWorkspaceOp;
+import com.liferay.ide.project.core.util.LiferayWorkspaceUtil;
+
+import java.io.File;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.sapphire.modeling.ProgressMonitor;
+import org.junit.Test;
+
+/**
+ * @author Andy Wu
+ */
+public class NewLiferayWorkspaceOpTests
+{
+
+    @Test
+    public void testNewLiferayWorkspaceOp() throws Exception
```

Comment:
Can you add some tests here that check for the validation messages for 'project already exists' etc?  Thanks.